calico_cni/constants.py

```diff
@@ -60,8 +60,8 @@
 ERR_CODE_GENERIC = 100   # Use this for all errors.
 
 # Policy modes.
-POLICY_MODE_ANNOTATIONS = "k8s-annotations"
-POLICY_MODE_DENY_INBOUND = "default-deny-inbound"
+POLICY_MODE_KUBERNETES_ANNOTATIONS = "k8s-annotations"
+POLICY_MODE_KUBERNETES = "k8s"
```

Comment:
Just a thought... do we want this to be `kubernetes` spelled out? Is that nicer or less nice?